### ISSUE TITLE
GitHub Desktop - Certificate revocation check failed

### ISSUE THREAD

**Issue by vadimfrey** [desktop/desktop#10175]
<img width="445" alt="GH_Desktop _error" src="https://user-images.githubusercontent.com/54370481/86974265-02f88600-c144-11ea-8478-42e0cf3bf49c.PNG">

When trying to fetch/push code while on corporate VPN I get the attached error message. Once VPN is disconnected I am able to fetch/push the code. How do I resolve this issue

**Comment by niik** [comment]
Hi @vadimfrey, sorry to hear you're having issues connecting while on your corporate VPN. Have you tried disabling revocation checks?

Could you upload a log file from GitHub Desktop so that I could get some more information about this issue? To access the log files go to the file menu in GitHub Desktop and select `Help` > `Show Logs in Finder (macOS) or Explorer (Windows)`.

The log files are created daily -- please upload a log file as an attachment from a day where you experienced the issue. 

Thanks!

**Comment by vadimfrey** [comment]
Hi @niik , I am cannot disable revocation checks since clicking "Disable revocation checks" button does not work for me. 

Attaching logs from yesterday and today as per your request - experience issue on both days
[2020-07-08.desktop.production.log](https://github.com/desktop/desktop/files/4898875/2020-07-08.desktop.production.log)
[2020-07-09.desktop.production.log](https://github.com/desktop/desktop/files/4898877/2020-07-09.desktop.production.log)

I also tried adding http.schannelCheckRevoke = false in Git global settings (via Eclipse) - see attached screenshot. This setting didn't help to resolve issue
<img width="478" alt="schannelCheckRevoke" src="https://user-images.githubusercontent.com/54370481/87078063-1a8e4800-c1f2-11ea-88cf-68b2cdad55d9.PNG">

**Comment by niik** [comment]
Hi again @vadimfrey. Could you try setting the `schannelCheckRevoke = false` setting on the `User Settings` level rather than the `System Settings`? I don't believe GitHub Desktop will respect the System level settings.

**Comment by vadimfrey** [comment]
Hmm, I don't see that Git option in my Advanced settings:
![image](https://user-images.githubusercontent.com/54370481/87090717-664aec80-c206-11ea-9230-b6aab617abe1.png)

Also, when I tried applying setting http.schannelCheckRevoke = false in User setting in Eclipse , I get an error message below:
![image](https://user-images.githubusercontent.com/54370481/87090911-c04bb200-c206-11ea-97a1-a5cb51edaf12.png)
I am guessing permission dependent

**Comment by vadimfrey** [comment]
Okay, this is great tip I actually just resolved the issue. 
1. I got access to this folder C:\Windows\System32\config\systemprofile using my local admin token
2. Once I got access to this folder I was able to apply setting http.schannelCheckRevoke = false setting on the User Settings level
3. Once that was done, I was able to click "Disable Revocation Checks" button in my GitHub desktop and this time the setting was applied and new Git option appeared on the the bottom of Advanced menu:

![image](https://user-images.githubusercontent.com/54370481/87091890-74017180-c208-11ea-8b61-8e5b9c9cac17.png)

As a result, I know get no errors while on corporate VPN. 

Thanks for your help, Markus!
